intake_xarray/opendap.py

```diff
@@ -38,7 +38,7 @@ class OpenDapSource(DataSourceMixin):
     """
     name = 'opendap'
 
-    def __init__(self, urlpath, chunks, auth="esgf", xarray_kwargs=None, metadata=None,
+    def __init__(self, urlpath, chunks, auth=None, xarray_kwargs=None, metadata=None,
```

Comment:
The docstring should be updated, and explicitly say that the default changed - that'll fix most for whom things suddenly stop working.

Reply:
Thanks for that hint. I've changed it.